src/Service/NuRaftLogSnapshot.cpp

```diff
@@ -401,7 +401,7 @@ void KeeperSnapshotStore::init(String create_time = "")
 {
     if (create_time.empty())
     {
-        Poco::DateTime now;
+        Poco::LocalDateTime now;
```

Comment:
please use time zone configured in config.xml

Reply:
Okay

Reply:
> please use time zone configured in config.xml
```
    if (config().has("timezone"))
    {
        const std::string config_timezone = config().getString("timezone");
        if (0 != setenv("TZ", config_timezone.data(), 1))
            throw Poco::Exception("Cannot setenv TZ variable");

        tzset();
        DateLUT::setDefaultTimezone(config_timezone);
    }
```
> tzset is often called at the beginning of a program or when the time zone needs to be changed. It ensures that the program uses the correct time zone information for functions like localtime, gmtime, and mktime.

Poco::LocalDateTime use local system timezone, use `gmtime_r` to get system time, it's same as use time zone configured in config.xml.
```
int Timezone::utcOffset()
{
	std::time_t now = std::time(NULL);
	struct std::tm t;
	gmtime_r(&now, &t);
	std::time_t utc = std::mktime(&t);
	return now - utc;
}
```

Poco::DateTime is always in UTC
```
	/// The date and time stored in a DateTime is always in UTC
	/// (Coordinated Universal Time) and thus independent of the
	/// timezone in effect on the system.
```